README.md

```diff
@@ -47,35 +45,47 @@ Workers are built in their own bin directories:
 * Client: `OtherWorkers/Interactive/client/bin`
 
 ## Cleaning the project
-Run `./clean.sh` to delete all build files, including worker binaries and any intermediate files generated during the build process.
+Run `./clean.sh` to delete all build files, including worker binaries and any intermediate files generated during the build process. Alternatively, run `git clean -xdf`.
 
-## Running the project
+## Running the project locally
 
-To launch a local instance of SpatialOS running the project,  run 
+To launch a local instance of SpatialOS running the project, run
 ```bash
-$ spatial alpha local launch --launch_config ./deployment.json
-``` 
+$ spatial alpha local launch
+```
 from the SpatialOS directory (or from any location by adding the `--main_config=\<path to spatialos.json\>` flag). This starts SpatialOS locally and runs the server workers `HelloWorker` and `DiceWorker`.
 
-Now you can connect game clients. You can find the client binaries in `OtherWorkers/Interactive/client/bin/x64/ReleaseWindows` (or `ReleaseMacOS` for Mac).
+You can optionally set the `--launch_config` flag to specify a filepath to the [launch configuration](docs/reference/launch-configuration.md) of your deployment. If the flag is not set, the spatial CLI will use launch configuration specified in the `launch_config` field of your [project configuration](docs/reference/project-configuration.md) as a fall back.
+
+As soon as your deployment is running, you can connect game clients to it. You can find the client binaries in `OtherWorkers/Interactive/client/bin/x64/ReleaseWindows` (or `ReleaseMacOS` for Mac).
```

Comment:
```suggestion
As soon as your deployment is running, you can connect client-workers to it. You can find the client-worker binaries in `OtherWorkers/Interactive/client/bin/x64/ReleaseWindows` (or `ReleaseMacOS` for Mac).
```

Assuming that "game client" == "client-worker" here, and that there's no reason why we'd specifically want to say 'game client".

Reply:
Done. Yes, he terms are synonymous.